frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
```

Comment:
Not sure about this, if you need to provide an element 120px from topmost point.
( ` has some top space which is equal to 120px`) IMHO the property `top` must be used instead of decreasing its viewport height.